### DISCUSSION TITLE
Last 10 earthquakes

### DISCUSSION THREAD

**Discussion by end-y** [acikkaynak/deprem-yardim-frontend#781] [Ideas]
It can show the last 10 earthquake data from the Kandilli Observatory.

I prepared API with php. It's live right now. But I used 000webhost for testing. If you approve of this, the site should have a strong infrastructure and the project. Leaders can guide me if any needs or improvements are needed.

By the way, there are other api options on Github. For your information.

Project link:
https://github.com/end-y/kandilli-api

**Comment by 0xCakin** [comment]
What kind of problem will this solve? How can it help victims or ER teams?
